core/core.djinni

```diff
@@ -25,3 +25,19 @@ LedgerCore = interface +c {
     # @return The integer version of the library
     static getIntVersion(): i32;
 }
+
+AllocationMetrics = interface +c {
+    static getObjectAllocations(): map<string, i32>;
+}
+
+DurationMetric = record {
+    # Total number of milliseconds
+    total_ms: i64;
+    # Number of records.
+    count: i32;
+}
+
+DurationMetrics = interface +c {
+    # Get all duration metrics
+    static getAllDurationMetrics(): map<string, DurationMetric>;
+}
```

Comment:
Should the metrics here be ported as well to hotfixs_3.5 branch ?

Reply:
I believe this is linked to the `_metrics` endpoint in the WD: https://github.com/LedgerHQ/ledger-wallet-daemon/blob/develop/src/main/scala/co/ledger/wallet/daemon/controllers/StatusController.scala#L53

This was very useful when we were working on NRT. But I don't think it has been integrated in DD nor Prometeus.
Any current plans on this topic @pollastri-pierre ?